src/components/Header/HeaderContact.js

```diff
@@ -66,9 +77,10 @@ const styles = StyleSheet.create({
     paddingVertical: SIZES.base,
     alignItems: 'center'
   },
-  content: {
-    padding: 10
-  },
+  content: (marginLeft) => ({
+    padding: 10,
+    marginLeft
+  }),
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `content` style has been updated to be a function that accepts a `marginLeft` parameter. This allows for dynamic styling based on the platform, as seen in the `renderHeader` function. This is a good practice as it allows for more flexible styling.
```diff
-  content: {
-    padding: 10
-  },
+  content: (marginLeft) => ({
+    padding: 10,
+    marginLeft
+  }),
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->